zdg/piece_collision.py

```diff
@@ -0,0 +1,23 @@
+import pygame
+import zdg.grid as Grid
+import zdg.event as Event
+from zdg.settings import CIRCLE_COLOR, BLOCK_SIZE
+
+
+class PieceCollision:
+    def __init__(self, event, grid):
+        self.event = event
+        self.grid = grid
+        self.cell = self.collides_with_piece()
+
+    def collides_with_piece(self):
+        for cell in self.grid.non_empty_cells():
+            if cell.collide_point(self.event.pos) == False:
```

Comment:
```suggestion
            if not cell.collide_point(self.event.pos):
```

Reply:
Ah yes this is more pythonic technically right. My brain processes == False better than not but I think its more normal to use if not so its a good replacement